### DISCUSSION TITLE
No Response from Github Support even after 14 Days

### DISCUSSION THREAD

**Discussion by AnujGoelInfoxen** [community/community#35307] [Other Feature Feedback, Questions, & Ideas]
I lost access to my org due to account recovery. I received an email that following the recovery process will make me out of org and would need to request again to get into it but i was the only admin of the org. I raised a ticket 14 days back with several followups but not getting any response. 

Still hoping to get it resolved soon,

**Comment by queenofcorgis** [comment]
Hi there! 

Thank you for being patient. Do you mind sending me your ticket number, I'm unable to escalate or be involved with the ticket, but I can confirm if it is in the right place! 

Lili

**Reply by AnujGoelInfoxen** [reply]
Hi @queenofcorgis , 

Thanks for your attention. 

Here is the ticket number : #1806286

Thanks Again.

**Reply by queenofcorgis** [reply]
Hi @parakhgoel - it looks like Support responded to you last night and sent you the proper steps needed. Please refer to that communication! Good luck!

**Reply by AnujGoelInfoxen** [reply]
Hi @queenofcorgis ,

Yes. Thanks. I will check and follow accordingly.
